packages/vite/src/vite.ts

```diff
@@ -103,8 +103,14 @@ export const bundle: NuxtBuilder['bundle'] = async (nuxt) => {
             composables: nuxt.options.optimization.keyedComposables,
           }),
           replace({
-            ...Object.fromEntries([';', '(', '{', '}', ' ', '\t', '\n'].map(d => [`${d}global.`, `${d}globalThis.`])),
```

Comment:
I would prefer to keep this as is, though it would be a nice improvement to split it out of the function.

Reply:
In my opinion, the question that should be asked here is whether or not this will change. Are there dynamic replacements planned to be added? If it's all static, this method doesn't hurt readability, it doesn't use iterations, and doesn't create intermediate arrays. I'll leave the decision up to you of course, but in my opinion this is an example of unmandatory iterations  that would just decrease runtime performance and increase memory usage (albeit, a little, but increase regardless) with no reasonably equivalent benefits.